misk-hibernate/src/test/kotlin/misk/hibernate/TransacterTest.kt

```diff
@@ -368,16 +367,15 @@ abstract class TransacterTest {
 
   @Test
   fun constraintViolationCausesTransactionToRollback() {
-    // Uniqueness constraints aren't reliably enforced on Vitess
-    assumeTrue(!transacter.config().type.isVitess)
-
     transacter.transaction { session ->
       session.save(DbMovie("Cinderella", LocalDate.of(1950, 3, 4)))
     }
     assertFailsWith<ConstraintViolationException> {
       transacter.transaction { session ->
         session.save(DbMovie("Beauty and the Beast", LocalDate.of(1991, 11, 22)))
-        session.save(DbMovie("Cinderella", LocalDate.of(2015, 3, 13)))
+        repeat(5) { // Repeat 5 times to trigger Vitess constraint on a single shard
```

Comment:
Should something in the stack be doing these retries automatically?

Reply:
This was just to ensure we get at least two writes on a single shard.  The unique constraint is limited to each shard.

Reply:
NVM, a single insert causes the constraint violation.  Not sure why I needed additional inserts before.